src/app/child-dev-project/children/children-list/list-attendance/list-attendance.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, Input } from "@angular/core";
+import { AttendanceMonth } from "../../../attendance/model/attendance-month";
+import { ChildrenService } from "../../children.service";
+import { Child } from "../../model/child";
+import { MediaChange, MediaObserver } from "@angular/flex-layout";
+import { OnInitDynamicComponent } from "../../../../core/view/dynamic-components/on-init-dynamic-component.interface";
+
+@Component({
+  selector: "app-list-attendance",
+  template: `
+    <app-attendance-block
+      *ngFor="
+        let att of attendanceList
+          | filterBy: { institution: filterBy }
+          | slice: 0:maxAttendanceBlocks
+      "
+      [attendanceData]="att"
+    ></app-attendance-block>
+  `,
+})
+export class ListAttendanceComponent implements OnInitDynamicComponent {
+  attendanceList: AttendanceMonth[] = [];
+  maxAttendanceBlocks: number = 3;
+
+  @Input() filterBy: string;
```

Comment:
These inputs would also benefit from a jsdoc. and maybe naming it `filterByInstitution` would be more fitting?